### PR TITLE
fix PostGenerationContext import path

### PR DESCRIPTION
**Pull request by skarzi**
This PR simply fixes the import path of `PostGenerationContext`.
Please note that I am not 100% familiar with the `pytest-factoryboy` source code, so it can be not fully working and ready for a production use solution.

---
This change is [Reviewable](https://reviewable.io/reviews/pytest-dev/pytest-factoryboy/109)

### CODE REVIEW

**Comment by skarzi** [comment]
### Feedback about `pre-commit.ci`

`pre-commit.ci` seems to not be properly configured yet, but personally, I am not a big fan of tools that creates commits automatically instead of giving developers some insights about the issues and allow them to decide if they want to create auto-commit or fix issues manually.
Auto-created commits could cause some troubles with rebasing/merging.

**Comment by skarzi** [comment]
Travis CI build run for this PR - https://travis-ci.org/github/pytest-dev/pytest-factoryboy/builds/752128907
I don't know why but for some reason GitHub mark Travis CI run as a failure and link `Details` to the oldest build (somehow canceled).

**Comment by youtux** [comment]
I think that the pre-commit bot was running before all the needed changes were merged.
Let's see how it goes from now on, if it's bothering the too much I will disable it.

File: pytest_factoryboy/compat.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+try:
+    from factory.declarations import PostGenerationContext
+except ImportError:
```
Comment (youtux) [inline pytest_factoryboy/compat.py:3]:
```suggestion
except ImportError:  # factory_boy < 3.2.0
```

**Review by youtux — CHANGES_REQUESTED** [review]
Can you add an entry to the CHANGES.rst about this?

**Comment by youtux** [comment]
Can you merge master again into your branch? I see an outdated CHANGES.rst in your changes.

**Review by youtux — APPROVED** [review]

**Comment by youtux** [comment]
Thank you!

**Pull request merged**